sideboard/tests/test_configuration.py

```diff
@@ -4,7 +4,23 @@
 import pytest
 from mock import Mock
 
-from sideboard.config import parse_config, get_config_root
+from sideboard.config import get_config_files, get_config_overrides, \
+    get_config_root, get_module_and_root_dirs, parse_config, uniquify
+
+
+def test_uniquify():
+    assert [] == uniquify([])
+    assert ['a', 'b', 'c'] == uniquify(['a', 'b', 'c'])
+    assert ['a', 'b', 'c', 'd', 'e'] == uniquify(['a', 'b', 'a', 'c', 'a', 'd', 'a', 'e'])
+    assert ['a'] == uniquify(['a', 'a', 'a', 'a', 'a', 'a', 'a', 'a'])
+
+
+@pytest.mark.skipif(
```

Comment:
I wanted to write a test that ensured `test-defaults.ini` was being loaded if `SIDEBOARD_CONFIG_OVERRIDES` was set in the environment (independently of monkey patching it after the tests have already begun).

However, I didn't want the test to fail if someone just ran the tests from the command line without setting the `SIDEBOARD_CONFIG_OVERRIDES`, so that's why it's skipped.

This was mostly a sanity check for my own benefit. I'll add a docstring explaining why this test exists.